indico/modules/events/papers/blueprint.py

```diff
@@ -41,6 +41,8 @@
                  management.RHAssignmentListCustomize, methods=('GET', 'POST'))
 _bp.add_url_rule('/manage/papers/assignment-list/judge', 'manage_judge_papers',
                  management.RHBulkPaperJudgment, methods=('POST',))
+_bp.add_url_rule('/manage/papers/assignment-list/download', 'manage_download_papers', management.RHDownloadPapers,
```

Comment:
So should I declare the blueprint to be available in both areas and move the RH declaration to a different controller e.g `paper.py` (similar to abstracts) instead of leaving it in `management.py`?
Because I think this action will be the same in management and display, isn't?